pmemcheck/pmemcheck.h

```diff
@@ -137,15 +146,15 @@ typedef
 
 /** Add a region of persistent memory, for which all operations will be
 * logged */
-#define VALGRIND_PMC_ADD_LOG_REGION(_qzz_addr,_qzz_len)       \
-    VALGRIND_DO_CLIENT_REQUEST_EXPR(0 /* default return */,          \
-                            VG_USERREQ__PMC_ADD_LOG_REGION,   \
+#define VALGRIND_PMC_ADD_LOG_REGION(_qzz_addr,_qzz_len)		    \
```

Comment:
There's something wrong with the whitespace at the end of the line (to align "\"). Either use spaces or tabs.